src/arrayMethodSort.js

```diff
@@ -4,8 +4,19 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const compare =
+      compareFunction || ((a, b) => (String(a) > String(b) ? 1 : -1));
```

Comment:
The default comparison function is correctly checking if `String(a)` is greater than `String(b)`, but it should also handle the case where they are equal by returning `0`. Consider using `(String(a) > String(b) ? 1 : (String(a) < String(b) ? -1 : 0))`.